doc/book/admin/logs.rst

```diff
@@ -56,21 +54,21 @@ this file is usually like:
        delaycompress
        create 0640 tarantool adm
        postrotate
-           /usr/bin/tarantoolctl logrotate `basename ${1%%.*}`
+           /usr/bin/tt logrotate `basename ${1%%.*}`
        endscript
    }
 
 If you use a different log rotation program, you can invoke
-``tarantoolctl logrotate`` command to request instances to reopen their log
+:ref:`tt logrotate <tt-logrotate>` command to request instances to reopen their log
 files after they were moved by the program of your choice.
 
 Tarantool can write its logs to a log file, ``syslog`` or a program specified
 in the configuration file (see :ref:`log <cfg_logging-log>` parameter).
 
-By default, logs are written to a file as defined in ``tarantoolctl``
-defaults. ``tarantoolctl`` automatically detects if an instance is using
+By default, logs are written to a file as defined in ``tt``
+defaults. ``tt`` automatically detects if an instance is using
```

Comment:
As I know, `tt` does not detect instance logging method. I'm not sure if `tarantoolctl` is able to do it. @LeonidVas do you know something about this functionality?

Reply:
I've rechecked this page. Seems like it needs a bigger update. Will rewrite it to reflect the ``tt`` usage and behavior.

Reply:
Rewrote the logs page, please check.